examples/models/widgets.py

```diff
@@ -39,15 +39,15 @@
 
 radio_button_group = RadioButtonGroup(labels=["Option 1", "Option 2", "Option 3"], active=0)
 
-text_input = TextInput(placeholder="Enter value ...")
+text_input = TextInput(placeholder="Enter value ...", title="<b>text</b>")
```

Comment:
That makes sense, definitely bigger scope of a change than what I was expecting to do though. `HTML` could inherit from `Markup` abstract model perhaps?